src/Widgets/calendar/Util.vala

```diff
@@ -20,6 +20,16 @@
  */
 
 namespace Util {
+    public enum Weekday {
```

Comment:
What's the reason for moving this from CalendarModel to Util?

Reply:
It's used on another method as being in Util, and it was spitting actual errors out. So I moved it.